lighthouse-core/audits/non-composited-animations.js

```diff
@@ -27,6 +27,16 @@ const UIStrings = {
     =1 {Unsupported CSS Property: {properties}}
     other {Unsupported CSS Properties: {properties}}
   }`,
+  /** Name of a compositor failure reason where the transform being animated depends on the box size. */
+  transformDependsBoxSize: 'Transform related property depends on box size',
+  /** Name of a compositor failure reason where the filter being animated may move pixels. */
+  filterMayMovePixels: 'Filter related property may move pixels',
+  /** Name of a compositor failure reason where the effect has a composite mode which is not replace. */
+  nonReplaceCompositeMode: 'Effect has composite mode other than "replace"',
+  /** Name of a compositor failure reason where the element already has an incompatible animation. */
+  incompatibleAnimations: 'Target has an incompatible animation',
```

Comment:
```suggestion
  incompatibleAnimations: 'Target has a second, incompatible animation',
```

or something to make it clear there's a *different* animation that is the problem? otherwise it just sounds like "It is invalid because it is invalid"